src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final Random RANDOM = new Random();
+
+    public Figure getRandomFigure() {
+        int figureNumber = RANDOM.nextInt(FIGURE_COUNT);
+        double randomSize = RANDOM.nextDouble() * 10 + 1;
+        Color randomColor = new ColorSupplier().getRandomColor();
+
+        switch (figureNumber) {
+            case 0:
+                return new Square(RANDOM.nextDouble() * 10 + 1,
+                        new ColorSupplier().getRandomColor());
```

Comment:
```suggestion
            case 0:
                return getRandomSquare();
```
lets add private methods to get each random figure